p2p/src/authenticated/actors/tracker/address.rs

```diff
@@ -18,15 +18,15 @@ pub struct Signature {
 }
 
 pub fn wire_peer_payload(peer: &Peer) -> Vec<u8> {
-    let mut payload = Vec::new();
+    let mut payload = Vec::with_capacity(peer.socket.len()+std::mem::size_of::<u64>());
     payload.extend_from_slice(&peer.socket);
     payload.extend_from_slice(&peer.timestamp.to_be_bytes());
     payload
 }
 
 pub fn socket_peer_payload(socket: &SocketAddr, timestamp: u64) -> (Vec<u8>, Vec<u8>) {
     let socket = bytes(socket);
-    let mut payload = Vec::new();
+    let mut payload = Vec::with_capacity(socket.len()+std::mem::size_of::<u64>());
     payload.extend_from_slice(&socket);
     payload.extend_from_slice(&timestamp.to_be_bytes());
```

Comment:
I wonder if we should switch this to `put_u64` (like we do in `mocks/application` above)?

I started using that pattern after writing this crate.

Reply:
Yeah I made a separate branch with those changes (`danlaine/danlaine-put-u64`). I'll put up a PR once this one gets merged.

Reply:
Appreciate it 🙏